backend-opal/src/server/portfolio.js

```diff
@@ -1,30 +1,14 @@
 const router = require('express').Router();
-const authService = require('./common/authorizationService');
+const authService = require('../middleware/authenticationService');
 
-// const portfolioList = [{ id: '1', userId: '1', symbol: 'MSFT', amount: 123 }];
 const stockService = require('./stockService');
 
-router.get('/', async (req, res) => {
-  const { authorization } = req.headers;
-  if (authService.checkAuthHeader(authorization)) {
-    const response = {
-      error: 'Authentication header is missing!'
-    };
+router.get('/', authService, async (req, res) => {
```

Comment:
@WenyuD since all our endpoints will be authenticated, you can just put it in the server:
`app.use(authService)`